lib/mumuki/laboratory/locales/es.yml

```diff
@@ -91,10 +91,13 @@ es:
   details: Detalles
   disabled_explanation: Estás intentando acceder a un recurso que fue deshabilitado o eliminado de forma permamente
   disabled_organization_explanation: Este recorrido ha finalizado
+  discussion_created_by: Creada por
   discussion_description_placeholder: Podés agregar información para detallar más tu consulta
   discussion_updated: Consulta actualizada
   new_discussion_message: Mensaje nuevo en %{title}
   discussions: Consultas
+  discussions_read: Leídas
+  discussions_unread: No leídas
```

Comment:
We had a single (unused) `read` key before this, but that proves a problem - we have a variety of options in other languages (`leído`, `leída`, `leídos` and `leídas` can't be grouped under a single `read`).

At first I even thought of
```
read:
  plural.f: Leídas
  plural.m: Leídos
  singular.f: Leída
  singular.m: Leído
```

But this still doesn't work. For example, if you wanted to show `Leído` as in `Mensaje leído` you'd look for `read.singular.m`, but `mensagem` is actually feminine in Portuguese - you can't assume genders here, either. Therefore, the smartest approach is the one which repeats a lot of code: for the cases I'm mentioning here, you need `discussion_read`, `discussions_read`, `message_read` and `messages_read`. That way there are no gender/cardinality problems when translating.

Reply:
I agree. It is fine now